### PR TITLE
update for valueof interface

### PR DESCRIPTION
**Pull request by ymhhh**
是因为如果slice的数据比较大，超出了sql的执行字符长度。那我将slice做了切分，并按认为可执行的长度[]interface{}传入到insertMulti中，xorm version: 769f6b3ae663248e8f1b1d8fecbe1eb26ac77ac7 是支持的。到了现在的版本不支持，加上reflect.ValueOf(Value.Interface())即可，本人已测过

### CODE REVIEW

**Comment by lunny** [comment]
Could you add some unit test?

**Comment by ymhhh** [comment]
sure, i would like to add some unit tests

File: session_insert_test.go [hunk 1]
```diff
@@ -26,3 +28,78 @@ func TestInsertOne(t *testing.T) {
...
+	var iLen int64
+	if sliceValue.Kind() != reflect.Slice {
+		return 0, fmt.Errorf("not silce")
+	}
+	iLen = int64(sliceValue.Len())
+	if iLen == 0 {
+		return
+	}
+
+	session := testEngine.NewSession()
```
Comment (lunny) [inline session_insert_test.go]: defer session.Close()

File: session_insert_test.go [hunk 2]
```diff
@@ -26,3 +28,78 @@ func TestInsertOne(t *testing.T) {
...
+		Id   int64  `xorm:"int(11) pk"`
+		Name string `xorm:"varchar(255)"`
+	}
+
+	assert.NoError(t, testEngine.Sync2(new(TestMulti)))
+
+	num, err := insertMultiDatas(1,
+		append([]TestMulti{}, TestMulti{1, "test1"}, TestMulti{2, "test2"}, TestMulti{3, "test3"}))
+	assert.NoError(t, err)
+	assert.Equal(t, 3, num)
```
Comment (lunny) [inline session_insert_test.go]: assert.EqualValues(t, 3, num)

**Comment by lunny** [comment]
Hi, could you rebase to master?

**Comment by ymhhh** [comment]
how to rebase to master? I don't know how!!!

**Comment by lunny** [comment]
maybe you should
```
git checkout master
git pull origin master
git push ymhhh master
```

**Comment by lunny** [comment]
Thanks!

**Pull request merged**